src/ch/imedias/rsccfx/model/connectionutils/RunRudp.java

```diff
@@ -184,8 +187,10 @@ public void run() {
         //model.getRemoteClientPort(), null, model.getIcePort()); alternative for starting Ice
         //on a fixed port (Could be an issue on p2p connection over NAT)
 
+       // rudpSocket = new ReliableSocket(model.getRemoteClientIpAddress().getHostAddress(),
+         //   model.getRemoteClientPort());
         rudpSocket = new ReliableSocket(model.getRemoteClientIpAddress().getHostAddress(),
-            model.getRemoteClientPort());
+            model.getRemoteClientPort(), null, model.getIcePort());
```

Comment:
see above: this is the other side, if one side fails and the other suceeds in ICE, we can still use p2p connection, but now we need the rudp server and client to switch positions...